tools/pd-simulator/simulator/cases/redundant_balance_region.go

```diff
@@ -35,23 +35,12 @@ func newRedundantBalanceRegion() *Case {
 	}
 
 	for i := 0; i < storeNum; i++ {
-		if i%2 == 1 {
-			simCase.Stores = append(simCase.Stores, &Store{
-				ID:        IDAllocator.nextID(),
-				Status:    metapb.StoreState_Up,
-				Capacity:  1 * units.TiB,
-				Available: 980 * units.GiB,
```

Comment:
we still need different `Available`, 980 * units.GiB and 1 * units.TiB

Reply:
Does this case still work? I test it and there are only balance leader operator.

Reply:
BTW, I think this can be solved through redistributing regions.

Reply:
Maybe we need to use different used size?

Reply:
Maybe we can put `ExtraUsedSpace` put into case to update store status

Reply:
> Maybe we can put `ExtraUsedSpace` put into case to update store status

I think so.

Reply:
I plan to simplify the case because we need to recompile it every time. Currently, it can also be done by using `ExtraUsedSpace` in the config file.

Reply:
If so, I think we should provide default value